tests/system/src/testapp7/java/mymod/myapp7/DataUrlWithModuleLayer.java

```diff
@@ -85,31 +91,24 @@ public void start(Stage primaryStage) throws Exception {
             + "<script src=\"data:application/javascript;base64," + checkJSEncoded + "\"></script>"
             + "</body>"
             + "</html>";
-
+        webview.getEngine().getLoadWorker().stateProperty().addListener(
+                new ChangeListener<State>() {
+                    public void changed(ObservableValue ov, State oldState, State newState) {
+                        String title = webview.getEngine().getTitle();
```

Comment:
Minor: you might want to move this inside the test for `SUCCEEDED` since it isn't valid until then.